internal/namespaces/instance/v1/custom_marshal.go

```diff
@@ -325,23 +325,25 @@ func (sg *customSecurityGroupResponse) MarshalHuman() (out string, err error) {
 	if err != nil {
 		return "", err
 	}
+	defaultInboundPolicy = terminal.Style(defaultInboundPolicy, color.Bold)
 
 	defaultOutboundPolicy, err := human.Marshal(sg.OutboundDefaultPolicy, nil)
 	if err != nil {
 		return "", err
 	}
+	defaultOutboundPolicy = terminal.Style(defaultOutboundPolicy, color.Bold)
 
 	inboundRulesContent, err := human.Marshal(inboundRules, nil)
 	if err != nil {
 		return "", err
 	}
-	inboundRulesView := terminal.Style(fmt.Sprintf("Inbound Rules (default policy %s):\n", defaultInboundPolicy), color.Bold) + inboundRulesContent
+	inboundRulesView := terminal.Style("Inbound Rules (default policy ", color.Bold) + defaultInboundPolicy + terminal.Style("):\n", color.Bold) + inboundRulesContent
```

Comment:
and here, it's not clear `defaultInboundPolicy` is already bold

Maybe remove `defaultInboundPolicy = terminal.Style(defaultInboundPolicy, color.Bold)` at l328, and here 

```
// comment explaning teh escaping problem
inboundRulesView := terminal.Style("Inbound Rules (default policy ", color.Bold) + 
    terminal.Style(defaultInboundPolicy,  color.Bold) + 
    terminal.Style("):", color.Bold) + 
    "\n" +  inboundRulesContent